Providing view of user board

igasm opened the pull request.

M-Aarsman commented on client/src/main/java/battleships/RootLayoutController.java:
```diff
@@ -0,0 +1,34 @@
+            borderPane.setLeft(initLayout);
+
+        } catch (IOException e) {
+            e.printStackTrace();
```
in future logger

M-Aarsman commented on client/src/main/java/battleships/game/Board.java:
```diff
@@ -0,0 +1,84 @@
+
+    List<Field> fields;
+
+    private Map<FieldState, Color> colorMap;
```
can be final

M-Aarsman commented on client/src/main/java/battleships/game/Board.java:
```diff
@@ -0,0 +1,84 @@
+    /**
+     * created for temporary use
+     */
+    void generateExample(){
```
this function can be divided to at least 4 private

M-Aarsman commented on client/src/main/java/battleships/logging/LoggingController.java:
```diff
@@ -97,5 +100,18 @@ private void initialize(LanguageLoadOption languageLoadOption) {
+
+    private void invalidLoggingDataAlert(){
+        Alert alert = new Alert(Alert.AlertType.WARNING);
+        alert.setTitle("Warning Dialog");
```
magic string -> Krzysztof

M-Aarsman commented on client/src/main/java/battleships/logging/LoggingController.java:
```diff
@@ -97,5 +100,18 @@ private void initialize(LanguageLoadOption languageLoadOption) {
+    private void invalidLoggingDataAlert(){
+        Alert alert = new Alert(Alert.AlertType.WARNING);
+        alert.setTitle("Warning Dialog");
+        alert.setHeaderText("Logging data is invalid");
```
magic string -> Krzysztof

M-Aarsman commented on client/src/main/java/battleships/logging/LoggingController.java:
```diff
@@ -97,5 +100,18 @@ private void initialize(LanguageLoadOption languageLoadOption) {
+        Alert alert = new Alert(Alert.AlertType.WARNING);
+        alert.setTitle("Warning Dialog");
+        alert.setHeaderText("Logging data is invalid");
+        alert.setContentText("...");
```
magic string -> Krzysztof

M-Aarsman commented on client/src/test/java/battleships/game/FieldTest.java:
```diff
@@ -0,0 +1,49 @@
+public class FieldTest {
+
+    @Test
+    public void givenPosition_whenCreatingField_itsPositionShouldBeEqualAsGiven(){
```
parametrize maybe

igasm commented:
I've changed variables to final, as you requested or just removed it. I split function generating ship into smaller ones. I've parametrized tests in FieldTest class. But I didn't extract strings that you mentioned to constants, because this are just messages.
